tests/main/proxy-no-core/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Ensure that the core.proxy.* settings are honored without core
+
+# only needs a test on classic
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
+
+restore: |
+    iptables -D OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
+    iptables -D OUTPUT -j REJECT -p tcp --dport http --reject-with tcp-reset
+    iptables -D OUTPUT -j REJECT -p tcp --dport https --reject-with tcp-reset
+
+execute: |
+    if ! command -v python3; then
+       echo "SKIP: need python3"
+       exit 0
+    fi
+    #shellcheck source=tests/lib/systemd.sh
+    . "$TESTSLIB"/systemd.sh
+    systemctl stop snapd.service snapd.socket
+    rm -rf /var/lib/snapd/state.json
+    systemctl start snapd.service snapd.socket
+
+    # only allow test user to connect to http/https
+    iptables -I OUTPUT -j REJECT -p tcp --dport http --reject-with tcp-reset
+    iptables -I OUTPUT -j REJECT -p tcp --dport https --reject-with tcp-reset
+    iptables -I OUTPUT -m owner --uid-owner "$(id -u test)" -j ACCEPT -p tcp
+    systemd-run --uid=test --unit tinyproxy -- python3 "$TESTSLIB/tinyproxy/tinyproxy.py"
+    # shellcheck source=tests/lib/systemd.sh
+    . "$TESTSLIB/systemd.sh"
+    wait_for_service tinyproxy
+
+    echo "Ensure normal install without proxy does not work"
+    if snap install core; then
+        echo "without a proxy core install should fail, test broken"
+        exit 1
+    fi
+
+    echo "Setup proxy config"
+    snap set core proxy.https=http://localhost:3128
+
+    echo "Ensure that snap install works even without an installed core to install core"
+    snap install core
+    check_journalctl_log 'CONNECT fastly.cdn.snapcraft.io' -u tinyproxy
```

Comment:
This failed in spread tests, it looks like missing helper "import" somewhere.

```
+ check_journalctl_log 'CONNECT fastly.cdn.snapcraft.io' -u tinyproxy
/bin/bash: line 89: check_journalctl_log: command not found
```

Reply:
Yeah, missing as well, thank you!